gno.land/cmd/gnoland/README.md

```diff
@@ -6,8 +6,88 @@
     $> cd ./gno/gno.land
     $> make install.gnoland
 
-## Run `gnoland` full node
+## Option 1: Run `gnoland` full node for local development
 
     $> gnoland start
 
 Afterward, you can interact with [`gnokey`](../gnokey) or launch a [`gnoweb`](../gnoweb) interface.
+
+
+## Option 2: Run a node and sync with a Proof of Authority (POA) network
+
+ - gnoland init
+
+ - Download genesis.json from a trusted source and save it to the root-dir/config directory.
+
+ - Get the peer's node id from the node you trust.
+
+ - Start the node
+
+       $> gnoland start --persistent "node_id@peer_ip_address:port" or add the persistent_peers value in the ./testdir/config/config.toml
+
+## Option 3: Run a node as a Proof of Authority validator starting from genesis state
+
+- Initialize the config and key files.
+
+      $> gnoland init
+
+- Return the node info; we will need it to add to validator info in the genesis.json
+
+      $> gnoland node
+
+      Address: "g14t47gv3v2z3pc23g3zr39mnc99w2cplp0jhqvv"
+      Pubkey: "E5IFULgXFdS49ILgvPmO3/8chuSWfbqw3zYXaNEP+60="
+
```

Comment:
In cosmos-sdk, this is 

```
gaiad tendermint show-node-id
gaiad tendermint show-address
```

I believe it should be 2 separate commands because most of the time, people use them with `my action to $(gnoland tm2 show-address)`